sinabs/backend/dynapcnn/weight_rescaling_methods.py

```diff
@@ -0,0 +1,56 @@
+# author    : Willian Soares Girao
+# contact   : williansoaresgirao@gmail.com
+
+import statistics
+from typing import Iterable
+
+import numpy as np
+
+
+def rescale_method_1(scaling_factors: Iterable[int], lambda_: float = 0.5) -> float:
+    """
+        This method will use the average (scaled by `lambda_`) of the computed re-scaling factor
+    for the pooling layer(s) feeding into a convolutional layer.
+
+    Arguments
```

Comment:
We need to keep consistency. So far we had the docstrings with `Parameters`, now we say `Arguments`. :sob: